app/src/cc/arduino/packages/formatter/AStyle.java

```diff
@@ -95,4 +107,4 @@ public String getMenuTitle() {
     return tr("Auto Format");
   }
 
-}
+}
```

Comment:
This removes a trailing newline at the end it seems, which is probably something your editor does automatically. Ideally, this change would not be included (by changing your editor settings to not make such whitespace changes, or using e.g. `git add -p` or similar to include only the relevant parts in the commit). Not a critical issue, though.

Reply:
Done...